metricbeat/module/traefik/health/data.go

```diff
@@ -0,0 +1,50 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package health
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstriface"
+)
+
+var (
+	schema = s.Schema{
+		"uptime": s.Object{
+			"sec": c.Float("uptime_sec"),
```

Comment:
Should this be Int?

Reply:
This value is a float in the response from Traefik's health API (http://localhost:8080/health) so I kept it the same here.

Reply:
In this case i would suggest we use milliseconds instead here, especially having https://github.com/elastic/elasticsearch/issues/31244 in mind. TBH for uptime I would also be fine to just convert it to Int. I don't think there is a benefit of having uptime in sub second detail.

Reply:
> I don't think there is a benefit of having uptime in sub second detail.

Yeah, good point. Will change to seconds.